CRM/Upgrade/Incremental/php/FiveTwentySix.php

```diff
@@ -79,16 +79,16 @@ public static function addNLBEOptionValue(CRM_Queue_TaskContext $ctx) {
       'is_active' => 1,
     ]);
     // Update the existing nl_NL entry.
-    \Civi\Api4\OptionValue::update()
-      ->addWhere('name', '=', 'nl_NL')
-      // Adding check against label in case they've customized it, in which
+    $sql = CRM_Utils_SQL::interpolate('UPDATE civicrm_option_value SET label = @newLabel WHERE option_group_id = #group AND name = @name AND label IN (@oldLabels)', [
```

Comment:
Maybe a silly question: Where is the interpolate function?

Reply:
Oh I see - it's part of this patch.

Reply:
Yeah, the function was originally two lines and then got a little longer for silly reasons.

I could break it out into a separate PR if that seems tidier.